src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -424,13 +424,9 @@ internal sealed override void DecodeWellKnownAttribute(ref DecodeWellKnownAttrib
             Debug.Assert(!attribute.HasErrors);
             Debug.Assert(arguments.SymbolPart == AttributeLocation.None);
 
-            if (attribute.IsTargetAttribute(this, AttributeDescription.NullableAttribute))
-            {
-                // NullableAttribute should not be set explicitly.
-                ((BindingDiagnosticBag)arguments.Diagnostics).Add(ErrorCode.ERR_ExplicitNullableAttribute, arguments.AttributeSyntaxOpt.Location);
-            }
+            ReportExplicitUseOfReservedAttributes(in arguments, ReservedAttributes.NullableAttribute);
```

Comment:
nit: It looks like we're now catching many more usages of reserved attributes on type parameters. Might be worth documenting as break.

Reply:
We are not. The behavior of this call is exactly the same as what was being done manually, just with less code duplication.